src/views/AgmExtension.vue

```diff
@@ -297,6 +297,12 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin,
     this.updateFilingData('add', FilingCodes.AGM_EXTENSION, undefined, undefined)
   }
 
+  /** Called just before this component is destroyed. */
+  beforeDestroy (): void {
+    // remove event handler
+    window.onbeforeunload = null
+  }
```

Comment:
I found a bug where, if you go to a local filing and change something then cancel out, refreshing or leaving the dashboard would display an "unsaved changes, are you sure you want to leave?" browser alert.

The bug was due to this event handler, which was set (line 266 above) but not cleared. So I added this here and in all other local filings that set the event handler.

(Ask if you want a more thorough explanation. Or Google it.)